extensions/azurePublishNew/src/components/provisioningWizards/footers/WizardFooterWithUserPersona.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import formatMessage from 'format-message';
+import styled from '@emotion/styled';
+import { DefaultButton, PersonaSize } from 'office-ui-fabric-react';
+
+import { UserInfo } from '../../../recoilModel/types';
+import { UserPersona } from '../../shared/userPersona/UserPersona';
+import { WizardNavigationState } from '../../shared/wizard/WizardNew';
+
+type ProvisonActionsStylingProps = {
+  showSignout: boolean;
+};
+const ProvisonActions = styled.div<ProvisonActionsStylingProps>((props) => ({
+  display: 'flex',
+  flexFlow: 'row nowrap',
+  justifyContent: props.showSignout ? 'space-between' : 'flex-end',
+}));
+
+const FooterButton = styled(DefaultButton)`
+  margin: '0 4px';
```

Comment:
I think each button needs to have a margin right of 8px except the last one? right?

Reply:
That's right.